src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -328,85 +328,87 @@ namespace OpenMS
       }
 
       // PrecursorCharge
-      !extractName(mytransition.precursor_charge, "PrecursorCharge", tmp_line, header_dict) &&
-      !extractName(mytransition.precursor_charge, "Charge", tmp_line, header_dict); // charge is assumed to be the charge of the precursor
+      void(!extractName(mytransition.precursor_charge, "PrecursorCharge", tmp_line, header_dict) &&
```

Comment:
what does this one do?

Reply:
avoids dozens of unused expression result warnings.